models/dtl/api/dtl_loc.py

```diff
@@ -0,0 +1,11 @@
+import requests
+import json
+import pandas as pd
+
+# Returns geometry rings for counties from ArcGIS API; returns 2000 rows - UPDATE to return all counties, calc (lat, long) Centroids
+
+def shp_get():
+    response = requests.get('https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/USA_Counties_Generalized/FeatureServer/0/query?where=1%3D1&outFields=NAME,STATE_NAME,STATE_FIPS,CNTY_FIPS,HSE_UNITS,SQMI,OBJECTID&outSR=4326&f=json')
```

Comment:
To deal with the long structure of URLs such as this, the `requests` module has a way to pass query paramenters (everything after the `query?`) [with code](https://requests.readthedocs.io/en/master/user/quickstart/#passing-parameters-in-urls). Try to convert to that.

I'm also concerned about the pseudo-random string after the base url. Is that an API key? If so, it probably shouldn't be checked in, and this URL should be [joined together](https://docs.python.org/3/library/urllib.parse.html?highlight=urlparse#urllib.parse.urljoin).

Reply:
Yeah - I haven't unpacked the ArcGIS API yet; neither the census one for that matter. But it does WORK; gimme the WIN!

Will open an issue:
 - Update ArcGIS API: pass params, remove limit, figure out what pseudo-random string is

Reply:
Covered by #31